CHANGELOG.md

```diff
@@ -23,6 +23,51 @@ This change was made in [pull request #3599: Enable new link styles by default](
 
 ### Breaking changes
 
+You must make the following changes when you migrate to this release, or your service might break.
+
+#### Update package file paths
+
+To support package dependencies, we’ve moved our source files from `src` into `package/src`. In preparation for additional build targets, our package is now compiled into `package/dist`.
+
+##### Node.js and other bundlers
+
+Depending on the bundler you use, you may also need to make changes to your configuration file or assets paths to replace `node_modules/govuk-frontend` with `node_modules/govuk-frontend/dist`.
```

Comment:
Thanks @claireashworth

I'm going to remove this section as @romaricpascal's suggested we just [tell everyone to use `dist/`](https://github.com/alphagov/govuk-frontend/pull/3498#discussion_r1194061783) (where applicable) rather than wonder whether it'll work or not

Reply:
@claireashworth Mind checking this looks alright?
https://github.com/alphagov/govuk-frontend/compare/04e5168b534140c5a751f7ded5141148d4d6ef7e..ece24c1e3a4fa6434655fe77c37ccd7f5458642d

I've followed the same approach as Nunjucks, Sass but for "**Node.js and other bundlers**"